doc/radosgw/notifications.rst

```diff
@@ -7,6 +7,10 @@ Bucket Notifications
 .. versionchanged:: Squid
    A new "v2" format for Topic and Notification metadata can be enabled with
    the :ref:`feature_notification_v2` zone feature.
+   Enabling the feature after an upgrade from an older version will trigger
```

Comment:
s/the feature/this feature/

But I can do this in a future PR.

LGTM.

Reply:
Looks like Yuval pushed that change.